app/presenters/address/manual.presenter.js

```diff
@@ -8,10 +8,19 @@
 /**
  * Formats data for the `address/{sessionId}/manual` page
  *
+ * @param {object} address - An object containing the saved address
+ *
  * @returns {object} - The data formatted for the view template
  */
-function go() {
-  return {}
+function go(address = {}) {
+  return {
+    pageTitle: 'Enter the address',
+    ...(address?.addressLine1 && { addressLine1: address.addressLine1 }),
```

Comment:
could these be 'addressLine1' with defaults of null ? 

Would be clearer in the return object what each is called ? 

Feel free to ignore but i don't think this is a common presenter return object pattern.

Reply:
are these better all on an address object ? 

address.addressLine1
address.postcode